src/abstract-classes/browser-plugin.js

```diff
@@ -51,6 +51,11 @@ class BrowserPlugin {
      * @param {string} [options.id]
      * @param {object} [options.launchOptions]
      * @param {string} [options.proxyUrl]
+     * @property {boolean} [useIncognitoPages]
+     *  If set to false pages use share the same browser context.
+     *  If set to true each page uses its own context that is destroyed once the page is closed or crashes.
```

Comment:
I would say "by default" instead "if set to false". Also `pages use share` does not make sense. Either pages use, or pages share